### PR TITLE
updated core dev team

### PR DESCRIPTION
**Pull request by luceos**

### CODE REVIEW

**Comment by franzliedke** [comment]
Thanks!

File: _docs/index.md [hunk 1]
```diff
@@ -27,7 +27,10 @@ To give Flarum a spin, go to the [Installation](http://flarum.org/docs/installat
...
 ## Core Team
 
-- Toby Zerner ([GitHub](http://github.com/tobscure), [Twitter](http://twitter.com/tobscure)) – Core Developer
-- Franz Liedke ([GitHub](http://github.com/franzliedke), [Twitter](http://twitter.com/franzliedke)) – Core Developer
-- Dominion ([GitHub](http://github.com/dcsjapan)) – i18n Lead
-- Daniël Klabbers ([GitHub](http://github.com/luceos), [Twitter](http://twitter.com/luceos)) – Community Manager
+- [Toby Zerner](http://discuss.flarum.org/u/Toby) ([GitHub](http://github.com/tobscure), [Twitter](http://twitter.com/tobscure)) – Core Developer
+- [Franz Liedke](http://discuss.flarum.org/u/Franz) ([GitHub](http://github.com/franzliedke), [Twitter](http://twitter.com/franzliedke)) – Core Developer
+- [Daniël Klabbers](http://discuss.flarum.org/u/luceos) ([GitHub](http://github.com/luceos), [Twitter](http://twitter.com/luceos)) – Project Manager
+- [Jordanjay29](http://discuss.flarum.org/u/jordanjay29) ([GitHub](http://github.com/jordanjay29) - Community Manager
```
Comment (edittler) [inline _docs/index.md:33]: Missing ")"

Comment (luceos) [reply _docs/index.md:33]: you're correct, are you able to do a pr for this bug?

Comment (franzliedke) [reply _docs/index.md:33]: Fixed: https://github.com/flarum/flarum.github.io/commit/612515e1a2c9e4dcc4da1e50c4c664b4b5057eeb

**Pull request merged**